data_architect_misc/data_transformer/transform_functions/apac_vietnam_transform_functions.py

```diff
@@ -39,11 +39,11 @@ def create_new_dataframe_from_given_sheet_names_and_add_media_type_column_using_
 
             # Create  RAW_MEDIA_TYPE column based on sheet names.
             temp_df[RAW_MEDIA_TYPE_COLUMN] = sheet
+            temp_df.dropna(subset=["Date", "Sector", "Brand"],inplace=True)
```

Comment:
Please rename this method (don't forget to update the config file!) like:
"create_new_dataframe_with_RAW_MEDIA_TYPE_column_from_input_file_with_multiple_sheets_dedicated_to_each_media_type"

Reply:
Done